src/CollectionsFrameWorkChallenge/LinkedListChallenge.java

```diff
@@ -0,0 +1,68 @@
+package CollectionsFrameWorkChallenge;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+public class LinkedListChallenge {
+
+    public static void main(String[] args) {
```

Comment:
As a future suggestion, if we have a feature/task/challenge to implement, it may be a good idea to have distinct methods for each bullet from the challenge.
We can have in the main method some collections declared that we will use, and after that, just call to simple methods that have desired logic.
We don't forget that the main() is the entry point in our application, but usually, we don't write the `boiler plate` code inside. We did this until now because we were at the beginning of the learning curve.

Reply:
Yeah that's actually a very good idea, just realized it's kind of hard for you also to also keep track and review each bullet from the ReadMe file. Will keep in mind this on the next challenges.